### DISCUSSION TITLE
a way to make the output into gltf/glb instead of PLY.

### DISCUSSION THREAD

**Discussion by Moaaz10** [openMVG/openMVG#2298] [General]
I need my files to have gltf/glb format. is there an option to change the format of the output? and if not do you know somewhere good to convert the files? (I am new to 3d modeling :) )
I want to use google modelviewer which only accepts gltf/glb format and converting the files online removes the textures from the mesh.

**Comment by pmoulon** [comment]
OpenMVG only output point cloud and camera position. Do you want this question for OpenMVS?

**Reply by Moaaz10** [reply]
Yes! I am sorry but I might be a little bit confused, This might not be the right place to ask? I have been using openMVG for the SFM and converting it to .mve format, then using OpenMVS programs there. As far as I understand MVE simplifies the process by having  MVG and MVS in one place.

The reason I don't use OpenMVE only with no conversions is that MVE doesn't extract my camera (SM-G9860) sensor information from the EXIF tags for some reason. neither did OpenMVG btw, which I found a solution that you suggested (ty!) to add the sensor information in the dataset file.

This system explained above works well for now. I am trying to use photogrammetry to make 3d models of furniture. if you have any advice to improve my little system here or improve the quality of my work would be appreciated!

-my problem here is that when I try to convert the ply files to gltf/glb using different websites it removes the textures and only leaves the mesh.
